pkgs/by-name/as/asmc/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "asmc";
+  version = "2.36.02";
```

Comment:
Needs a note on how the project is versioned oddly.
Not using tags and having several commits with the same "version" message

Reply:
Maybe this should be versioned like `2.36.02-unstable-2024-10-01` since it is off a commit.

If there was only ever one commit with the version title I could be convinced to leave it but I think it does need to follow this format to denote it's versioned a bit oddly

Reply:
Ugh yeah I didn’t notice that.